### DISCUSSION TITLE
Can you provide a one-click deployment command in Linux?

### DISCUSSION THREAD

**Discussion by YyJyyy** [danny-avila/LibreChat#2708] [Feature Requests & Suggestions]
> Original: 
> 在linux中可以进行一键部署，是否可以出这样的文档，或者在linux中部署的时候文档写的详细一些。

Can there be a one-click deployment in Linux? Could you provide such documentation, or provide more detailed documentation for deployment in Linux?

**Comment by danny-avila** [comment]
I had to translate your query. Yes, at least for digitalocean, I plan on writing a script like this.
